formatters/rust.py

```diff
@@ -286,7 +286,7 @@ def get_summary(self):
 class StdVecDequeSynthProvider(RustSynthProvider):
     def update(self):
         self.ptr = read_unique_ptr(gcm(self.valobj, 'buf', 'ptr'))
-        self.cap = gcm(self.valobj, 'buf', 'cap').GetValueAsUnsigned()
+        self.cap = gcm(self.valobj, 'buf', 'cap', '0').GetValueAsUnsigned()
```

Comment:
won't this now fail on existing code? if the version of the standard library is prior to the change to a cap tuple, we will have the opposite problem, no?

Reply:
I'll add a test to check if it's a pod or not, similar to the end part.

Reply:
The new implementation addresses your worries I think, is this okay with you?

Reply:
yep LGTM